setup.php

```diff
@@ -143,13 +143,24 @@ function cloneClicked() {
 <span class="text">
 
 <?php
+// Checks if the server's PHP version and the client's browser are compatible with OpenEMR:
+require_once(dirname(__FILE__) . "/common/compatibility/checker.php");
```

Comment:
skip the above idea also :)
Would likely just cause pain (for example, it would define a broken xl() function)